internal/bankid/endpoints.go

```diff
@@ -153,6 +153,71 @@ func (a *API) WatchForChange(ctx context.Context, orderRef string) chan WatchRes
 	return watch
 }
 
+type Change struct {
+	CollectResponse
+	Err error
+}
+
+func (a *API) WatchForChangeV2(ctx context.Context, orderRef string) (<-chan Change, error) {
```

Comment:
Should we perhaps name this something different to highlight their intended purposes?